mitol-django-common/pyproject.toml

```diff
@@ -24,34 +24,39 @@ include = [
 
 [tool.poetry.dependencies]
 python = "~3.6 || ~3.7 || ~3.8"
-Django = "^2.2.12"
+Django = "^2.2.12 || ~3.0 || ~3.1"
 psycopg2-binary = "^2.8.3"
 requests = "^2.20.0"
 
-[tool.poetry.dev-dependencies]
```

Comment:
Do you know where you specify package dependencies for poetry when it uploads to PyPI? I'm wondering if the below dev dependencies would be listed as dependencies if we remove this line

Reply:
they end up being grouped under extras - I had to switch them over to this in order for it to work with tox becuase poetry's concept of `dev-dependencies` is a non-standard thing, so tox won't pick it up (and they've indicated on github issues they don't desire to support this special case)